src/selectors/visibleBreakpoints.js

```diff
@@ -0,0 +1,30 @@
+import { getSelectedSource } from "../reducers/sources";
+import { isGeneratedId } from "devtools-source-map";
+
+function getLocation(breakpoint, isGeneratedSource) {
+  return isGeneratedSource
+    ? breakpoint.generatedLocation || breakpoint.location
+    : breakpoint.location;
+}
+
+export default function getVisibleBreakpoints(state: OuterState) {
```

Comment:
is this so that we can check if a breakpoint is also represented in a generated file? can we use it in all cases?